Library/Homebrew/extend/ARGV.rb

```diff
@@ -216,7 +216,8 @@ def build_from_source?
   end
 
   def build_all_from_source?
-    !!ENV["HOMEBREW_BUILD_FROM_SOURCE"]
+    return true if ENV["HOMEBREW_BUILD_FROM_SOURCE"]
```

Comment:
To make sure, `!value.nil?` will only work for `ENV` variables, since those can't be `false`, or can they?